src/mbgl/style/layers/symbol_layer.cpp

```diff
@@ -169,7 +169,7 @@ void SymbolLayer::setIconSize(PropertyValue<float> value) {
     if (value == getIconSize())
         return;
     impl->layout.iconSize.set(value);
-    impl->observer->onLayerLayoutPropertyChanged(*this);
+    impl->observer->onLayerLayoutPropertyChanged(*this, true);
```

Comment:
> symbol_layer.cpp is a generated file, so this change will get overwritten.

Of course.. I was a bit too hasty there.

> Instead of a boolean, how about a const char * property, with name from style spec

Makes sense, I'll start on that now.